utils/swipe.js

```diff
@@ -0,0 +1,97 @@
+export default class Swipe {
+
+  constructor(elm) {
+    this.handleMove = this.handleMove.bind(this);
+    this.handleStart = this.handleStart.bind(this);
+    this.handleEnd = this.handleEnd.bind(this);
+
+    this.$elm = typeof elm === 'string' ? document.querySelector(elm) : elm;
+    this.hasTouch = 'ontouchstart' in window;
+    this.bindEvents();
+  }
+
+  bindEvents() {
+    if (this.hasTouch) {
+      this.$elm.addEventListener('touchstart', this.handleStart, false);
```

Comment:
I've just added the code below. It'll be easier when changed from class to component.